csharp/ql/test/experimental/ir/ir/PrintAst.expected

```diff
@@ -214,10 +214,10 @@ collections.cs:
 #   12|     4: [BlockStmt] {...}
 #   13|       0: [LocalVariableDeclStmt] ... ...;
 #   13|         0: [LocalVariableDeclAndInitExpr] Dictionary<Int32,MyClass> dict = ...
-#   13|           -1: [TypeMention] Dictionary<Int32, MyClass>
+#   13|           -1: [TypeMention] Dictionary<int, MyClass>
```

Comment:
`TypeMention` nodes are rendered by calling `toStringWithTypes()`, which now call `toStringWithTypes()` on type arguments. And `toStringWithTypes()` is overridden in `IntType`. Should we override `toString()` too? It is currently returning `Int32`.

Reply:
I don't think we should change `toString`.